kotlin-react-router-dom/src/main/kotlin/react/router/dom/routingDsl.kt

```diff
@@ -81,34 +81,18 @@ fun RBuilder.route(
     )
 }
 
-fun <T : Props> RBuilder.route(
-    vararg path: String,
-    exact: Boolean = false,
-    strict: Boolean = false,
-    render: RBuilder.(RouteResultProps<T>) -> Unit,
-) {
-    child<RouteProps<T>, Route<T>> {
-        attrs {
-            this.path = path
-            this.exact = exact
-            this.strict = strict
-            this.render = { props -> buildElements({ render(props) }) }
-        }
-    }
-}
-
 fun RBuilder.route(
     vararg path: String,
     exact: Boolean = false,
     strict: Boolean = false,
-    render: Render,
+    render: RBuilder.(RouteResultProps) -> Unit,
```

Comment:
RHandler?

Reply:
`typealias RHandler<P> = RElementBuilder<P>.() -> Unit`
`RHandler` does not do that is needed here